test/e2e/managedseed/create_rotate_delete.go

```diff
@@ -131,7 +132,41 @@ var _ = Describe("ManagedSeed Tests", Label("ManagedSeed", "default"), func() {
 			Eventually(func() error {
 				return triggerGardenletKubeconfigRotationViaSecret(ctx, shootClient.Client(), seed.Status.Gardener.Name)
 			}).Should(Succeed())
-			verifier.After(ctx)
+			verifier.After(ctx, true)
+		}
+
+		By("Trigger gardenlet kubeconfig auto-rotation by reducing kubeconfig validity")
+		{
+			By("Update kubeconfig validity settings")
+			Eventually(func() error {
+				// This configuration will cause the gardenlet to automatically renew its client certificate roughly every 60s.
+				// The actual certificate is valid for 15m (even though we specify only 10m here) because kube-controller-manager
+				// backdates the issued certificate, see https://github.com/kubernetes/kubernetes/blob/252935368ab67f38cb252df0a961a6dcb81d20eb/pkg/controller/certificates/signer/signer.go#L197.
+				// ~40% * 15m =~ 6m. The jittering in gardenlet adds this to the time at which the certificate was issued and then
```

Comment:
```suggestion
				// ~40% * 15m =~ 6m. The jittering in gardenlet adds this to the time at which the certificate became valid and then
```
Nit: The `issue date` is now, but the `start of validity date` is 5 minutes in the past.